db/seeds/20180108175137-admin-user.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+require('dotenv').config();
+var bcrypt = require('bcrypt');
+var username = process.env.ADMIN_USERNAME || 'admin';
+
+module.exports = {
+  up: function (queryInterface, Sequelize) {
+    var password = process.env.ADMIN_PASSWORD || 'development';
+    var now = new Date().toISOString();
+    return bcrypt.hash(password, 10).then(function(encrypted_password) {
+      return queryInterface.bulkInsert('users', [{
+        name: username,
+        encrypted_password: encrypted_password,
+        superuser: true,
+        email: process.env.ADMIN_EMAIL,
```

Comment:
Is there a fallback for `email` if it’s not provided in `.env`?

Reply:
No, I thought I'd leave it blank and let it fail if one is not provided, since it really needs to be the actual email address of the person setting up the repo.

Reply:
IMO, the password could possibly left blank. It's not a required field as a user is also able to log in using OAuth. Perhaps you should have a conditional here that checks whether the password has been provided, and returns the `bcrypt.hash` promise or skips straight to the `queryInterface.bulkInsert` call.

Reply:
Sure, I'll add that in.